test/Common/TestOrchestrations.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.Azure.WebJobs.Host.TestCommon;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Primitives;
+using static Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests.DurableTaskEndToEndTests;

```

Comment:
Not used?

Reply:
I added an orchestration that uses the ComplexType that is defined in that file. Should I pull it out into its own file?